config.js

```diff
@@ -24,7 +24,7 @@ module.exports = {
     passwordSalt: '$2a$10$l64QftVz6.7KR5BXNc29IO',
     port: 443,
     httpMode: false,
-    root: 'https://test.com',
+    root: 'https://localhost',
```

Comment:
I would rather not change the default config.js for the tests. Just start the server with a custom config for the tests.